recipes-extended/onl/onl_git.bb

```diff
@@ -62,3 +81,11 @@ ONL_PLATFORM_SUPPORT:x86-64 = " \
     x86-64-delta-ag5648-r0 \
     x86-64-delta-ag7648-r0 \
 "
+
+# common vendor kernel modules vendors to build
```

Comment:
I suppose this was meant to be:
```suggestion
# common vendor kernel modules to build
```

Reply:
No, that was intentional. This is the list of vendors for which we build the (common) vendor modules (onl package is named `onl-vendor-${vendor}-modules`).

Reply:
Alright, I understand what you mean, but it's still weird. We are building modules, not vendors, but the comment suggests otherwise. Can you rephrase the comment in a way that is easier to understand?

Reply:
"vendors for which we build common vendor modules"?

Reply:
Works for me.

Reply:
WOHOOOOO!!!

Reply:
Updated